src/m365/aad/commands/user/user-hibp.spec.ts

```diff
@@ -64,11 +64,6 @@ describe(commands.USER_HIBP, () => {
     assert.notStrictEqual(command.description, null);
   });
 
-  it('fails validation if userName and apiKey is not specified', async () => {
```

Comment:
Why are we removing all these tests? They are valid cases that we should check for.

Reply:
Thanks for the review Waldek. 

If I have understood correctly, if we have marked the [options are mandatory](https://github.com/pnp/cli-microsoft365/blob/eb196180d466c52eb541e3b12a5427d41462ecb6/src/m365/aad/commands/user/user-hibp.ts#L14-L15), the validation would be done on the base class. That being the case, even the testing for the **required logic** would be done on the base class and not the command implementation and we may not need to have the test cases defined in the commands. 
It is quite possible that I might have totally understood the updated testing wrong since I am super rusty in the command creation logic done now Pardon me if that is the case.

My understanding is totally based on the command refactoring discussion [done here](https://github.com/pnp/cli-microsoft365/issues/3218#issuecomment-1157292252). 

Sorry that If I have understood the current process totally wrong.

Reply:
A better way would be to mock the setting so that we don't prompt for the missing options.

Reply:
Sure Waldek. Thanks for the direction. I will make the PR as draft and relook into it.